airflow/providers/google/marketing_platform/hooks/analytics.py

```diff
@@ -66,10 +66,43 @@ def list_accounts(self) -> List[Dict[str, Any]]:
             # start index has value 1
             request = accounts.list(start_index=len(result) + 1)
             response = request.execute(num_retries=self.num_retries)
-            result.extend(response.get('items', []))
+            result.extend(response.get("items", []))
             # result is the number of fetched accounts from Analytics
             # when all accounts will be add to the result
             # the loop will be break
             if response["totalResults"] <= len(result):
                 break
         return result
+
+    def list_ad_words_links(
+        self, account_id: str, web_property_id: str
+    ) -> List[Dict[str, Any]]:
+        """
+        Lists webProperty-Google Ads links for a given web property.
+
+        # :param account_id: ID of the account which the given web property belongs to.
+        # :type account_id: str
+        # :param web_property_id: Web property ID to retrieve the Google Ads links for.
+        # :type web_property_id: str
+
```

Comment:
It would be great to describe what are the results. What do they contain? Link to the doc in Google docs is enough but an example would be great.